plugin/pkg/global/finalizerremoval/admission.go

```diff
@@ -80,8 +80,10 @@ func (f *FinalizerRemoval) ValidateInitialization() error {
 	return nil
 }
 
-// Admit ensures that finalizers from objects can only be removed if they are not needed anymore.
-func (f *FinalizerRemoval) Admit(_ context.Context, a admission.Attributes, _ admission.ObjectInterfaces) error {
+var _ admission.ValidationInterface = &FinalizerRemoval{}
```

Comment:
See https://github.com/gardener/gardener/pull/12759#discussion_r2290129629:

```suggestion
var _ admission.ValidationInterface = (*FinalizerRemoval)(nil)
```